vitest.browser.config.mts

```diff
@@ -30,7 +30,12 @@ const config: UserConfig = {
     }),
   ],
   optimizeDeps: {
-    exclude: ["fsevents", "path-scurry", "@vitest/coverage-istanbul"],
+    exclude: [
+      "fsevents",
+      "path-scurry",
+      "@vitest/coverage-istanbul",
+      "chromium-bidi",
```

Comment:
Added this based on [this bug](https://github.com/microsoft/playwright/issues/33031) and [this comment](https://github.com/microsoft/playwright/issues/33031#issuecomment-2405403388).